src/corelibs/U2View/src/ov_sequence/PanViewRows.cpp

```diff
@@ -28,6 +28,8 @@
 
 namespace U2 {
 
+const QString PVRowData::RESTRICTION_SITE_NAME = QObject::tr("Restriction Site");
```

Comment:
You haven't updated the translation file

Reply:
Translations will be updated massively before release